libpod/events/config.go

```diff
@@ -36,6 +36,18 @@ type Event struct {
 	Time time.Time
 	// Type of event that occurred
 	Type Type
+
+	Details
```

Comment:
Why do we need the embedded struct? Why not just put Attributes in here?

Reply:
I was following the docker event structure to some degree.

Reply:
I think this is overcomplicating matters, I don't see any case where we'll reuse this sub-struct.